docs/develop/php/index-php.mdx

```diff
@@ -54,7 +45,7 @@ Ensure that you are able to use the following Redis command to connect to the Re
  redis-cli -h localhost -p 6379
  localhost>
 ```
-
+Now you should be able to perform CRUD operation with Redis keys. The above redis client command might require password if you have setup authentication in your redis configuration file. If Redis password is not set, then it will perform the default connection to Redis server. You can play around inserting data to Redis via SET and then fetching it back via GET command.
```

Comment:
```suggestion
Now you should be able to perform CRUD operations with Redis keys. The above Redis client command might require password if you have setup authentication in your Redis configuration file. If a Redis password is not set, then it will perform the default connection to Redis server. You can play around inserting data to Redis using `SET` and then fetching it back with the `GET` command.
```